clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -319,10 +319,11 @@ Changes in existing checks
   diagnostic.
 
 - Improved :doc:`readability-implicit-bool-conversion
-  <clang-tidy/checks/readability/implicit-bool-conversion>` check
-  by adding the option `UseUpperCaseLiteralSuffix` to select the
-  case of the literal suffix in fixes and fixing false positive for implicit
-  conversion of comparison result in C23.
+  <clang-tidy/checks/readability/implicit-bool-conversion>` check by adding the
+  option `UseUpperCaseLiteralSuffix` to select the case of the literal suffix in 
+  fixes and fixing false positive for implicit conversion of comparison result in 
+  C23 , and by adding the option `CheckConversionsToBool` or 
```

Comment:
```suggestion
  C23, and by adding the option `CheckConversionsToBool` or 
```

Reply:
Done Thanks.

Reply:
Hey, What needs to be done further? Can you please help me out?

Reply:
Please fix Clang-format complains.

Reply:
Done Thanks.